cpp/CMakeLists.txt

```diff
@@ -90,6 +90,12 @@ option(
 mark_as_advanced(CUDF_BUILD_STREAMS_TEST_UTIL)
 option(CUDF_STATIC_LINTERS "Enable static linters during compilation" OFF)
 
+option(
+  CUDF_KVIKIO_REMOTE_IO
+  "Enable remote IO (e.g. AWS S3) support through KvikIO. If disabled, cudf-python will still be able to do remote IO."
```

Comment:
If this is not enable, the C++ code will not support it thus I don't think Python will be able to do it.

Reply:
Updated: 
```cmake

option(
  CUDF_KVIKIO_REMOTE_IO
  "Enable remote IO (e.g. AWS S3) support through KvikIO. If disabled, cudf-python will still be able to do remote IO through fsspec."
  ON
)

```